File: templates/tool-template/examples/basic.rb
#!/usr/bin/env ruby
# frozen_string_literal: true

# Minimal copy-paste example. Must run in <5 min.
ENTRY = File.join(File.expand_path('..', __dir__), 'main.rb')
exit(system(RbConfig.ruby, ENTRY, '--help') ? 0 : 1)

File: templates/tool-template/examples/basic.py
#!/usr/bin/env python3
"""Minimal copy-paste example. Must run in <5 min."""
import subprocess
import sys
from pathlib import Path

ENTRY = Path(__file__).resolve().parent.parent / "main.py"
raise SystemExit(subprocess.call([sys.executable, str(ENTRY), "--help"]))
